joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -1165,11 +1172,32 @@ bool JointTrajectoryController::validate_trajectory_msg(
 
     const size_t joint_count = trajectory.joint_names.size();
     const auto & points = trajectory.points;
-    if (
+    // TODO(anyone): This currently supports only position, velocity and acceleration inputs
```

Comment:
```suggestion
    // This currently supports only position, velocity and acceleration inputs
```
this isn;t a todo but a simple comment